src/tools/pkcs11-tool.c

```diff
@@ -227,6 +233,9 @@ static const char *option_help[] = {
 	"Derive a secret key using another key and some data",
 	"Derive ECDHpass DER encoded pubkey for compatibility with some PKCS#11 implementations",
 	"Specify mechanism (use -M for a list of supported mechanisms)",
+	"Specify hash algorithm used with generic RSA-PSS signature",
+	"Specify MGF (Message Generation Function) used for RSA-PSS signatures (possible values are MGF1-SHA1 to MGF1-SHA512)",
+	"Specify how many bytes should be used for salt in RSA-PSS signatures (default 0)",
```

Comment:
I think default here should be the same as OpenSSL. And OpenSSL uses `rsa_pss_saltlen:-1` as its RSA-PSS default.